tests/kernel/mem_protect/futex/prj.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_ZTEST=y
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_MP_NUM_CPUS=1
+CONFIG_LEGACY_TIMEOUT_API=y
```

Comment:
why does the futex test need this?

Reply:
It passes its own timeout value around via pointer arguments while assuming it's an integer, and would have been a minor port to the logic.